package.json

```diff
@@ -44,5 +44,24 @@
   "repository": {
     "type": "git",
     "url": "https://github.com/nationalbankbelgium/stark.git"
+  },
+  "devDependencies": {
+    "husky": "0.14.3",
+    "lint-staged": "7.0.0",
+    "prettier": "1.10.2",
+    "stylelint": "8.2.0",
```

Comment:
We use now version 9.1.1 of Stylelint, could you change this dependency and check if it still works?

We still need to check whether the dependencies to Stylelint and TSLint are to be placed here or in the build package

Reply:
The goal is to add a hook "precommit" in all the project. For doing that, we need those dependencies in the root.
In any case, we should upgrade the dependencies to the latest versions